Makefile

```diff
@@ -199,7 +199,7 @@ fmt-check:
 
 .PHONY: test
 test:
-	GO111MODULE=on $(GO) test -mod=vendor -tags='sqlite sqlite_unlock_notify' $(PACKAGES)
+	GO111MODULE=on $(GO) test -mod=vendor -tags='sqlite sqlite_unlock_notify' $(PACKAGES) && echo PASS
```

Comment:
This seems unrelated to PR

Reply:
it Is - thought this make `test` and `test-sqlite` more similar

Reply:
@mrsdizzie ^

Reply:
I noticed this, although unrelated I wasn't against the slight improvement here. I suspect though that this is a big enough PR that this is probably unnecessary noise.